fastlane/fastlane
# [spaceship] fix live family edit

### Checklist

### Motivation and Context
It seems that there are currently multiple issues when it comes to updating live subscriptions on the App Store. 

If you try to update a `subscription family` for example, `spaceship` will fail with the following error:
> You must have at least one approved localization on this subscription.

Even though the user can simply go to App Store Connect and change the name or any other setting there. After some investigation I found that `spaceship` was missing an ID in the `details` object. Once added one can edit both live & draft subscription families.
 
### Testing Steps
```
bundle exec irb
require "spaceship"
Spaceship::Tunes.login("test@test.com", "•••••••••••••••••")
app = Spaceship::Tunes::Application.find(123456789)
edit = app.in_app_purchases.families.all.first.edit
edit.name = "renamed"
edit.save!
```

The same code above should fail on any existing `fastlane` version with the above mentioned error for any approved & live subscription families/groups.
